src/core_plugins/timelion/server/series_functions/es/index.js

```diff
@@ -28,44 +29,65 @@ export default new Datasource('es', {
       name: 'q',
       types: ['string', 'null'],
       multi: true,
-      help: 'Query in lucene query string syntax'
+      help: i18n.translate('timelion.help.functions.es.args.qHelpText', {
+        defaultMessage: 'Query in lucene query string syntax',
+      }),
     },
     {
       name: 'metric',
       types: ['string', 'null'],
       multi: true,
-      help: 'An elasticsearch metric agg: avg, sum, min, max, percentiles or cardinality, followed by a field.' +
-        ' E.g., "sum:bytes", "percentiles:bytes:95,99,99.9" or just "count"'
+      help: i18n.translate('timelion.help.functions.es.args.metricHelpText', {
+        defaultMessage:
+          'An elasticsearch metric agg: avg, sum, min, max, percentiles or cardinality, followed by a field. \
+E.g., "sum:bytes", "percentiles:bytes:95,99,99.9" or just "count"',
+      }),
```

Comment:
Please add the following description: `avg, sum, min, max, percentiles and cardinality are keywords in the expression and must not be translated. Also don't translate the examples.`